src/vfocus.js

```diff
@@ -300,3 +300,13 @@ VFocus.prototype.children = function(){
 VFocus.prototype.addChild = function(child){
   this.vNode.children.push(child);
 };
+
+VFocus.prototype.indexOf = function(focus){
```

Comment:
Okay, i think there's some confusion here.  This `indexOf` method returns the index of a node within the VFocus' children. However you comment is good but I think there should be an `index()` method which returns the index of `this` relative to the parent?